src/pages/FAQ/FAQ.jsx

```diff
@@ -1,11 +1,19 @@
 import React from "react";
 import Box from "@mui/material/Box";
```

Comment:
:pick: combine the `Box` and `Typography` imports in one line

Reply:
Addressed:  e00c467d80c1180d1b943bf251488edfd4ec6854